### PR TITLE
Update cluster.rst

### PR DESCRIPTION
**Pull request by wiersgallak**

### CODE REVIEW

File: source/administration/upgrade.rst [hunk 1]
```diff
@@ -130,6 +130,10 @@ If they still show yellow, then you need to trigger a config propagation across
 
 After the server is upgraded, users might need to refresh their browsers to experience any new features.
 
+.. note::
+
+  We only support a one version difference between the server versions when performing a rolling upgrade. Running two different versions of Mattermost in your cluster should not be done outside of an upgrade scenario.
```
Comment (grundleborg) [inline source/administration/upgrade.rst:135]:
```suggestion
  We only support a one minor version difference between the server versions when performing a rolling upgrade. Running two different versions of Mattermost in your cluster should not be done outside of an upgrade scenario. 

```

Maybe this needs to have an example too: e.g.
* 5.27.1 + 5.27.2 is good
* 5.26.4 + 5.27.1 is good
* 5.25.5 + 5.27.0 is bad

File: source/deployment/cluster.rst [hunk 2]
```diff
@@ -343,6 +343,10 @@ A service interruption is not required for security patch dot releases of the Ma
 
 You can apply updates during a period when the anticipated load is small enough that one server can carry the full load of the system during the update.
 
+.. note::
+
+  We only support a one version difference between the server versions when performing a rolling upgrade. Running two different versions of Mattermost in your cluster should not be done outside of an upgrade scenario.
```
Comment (grundleborg) [inline source/deployment/cluster.rst:348]: As above

**Comment by wiersgallak** [comment]
@grundleborg Updated with your feedback.

File: source/administration/upgrade.rst [hunk 3]
```diff
@@ -130,6 +130,11 @@ If they still show yellow, then you need to trigger a config propagation across
 
 After the server is upgraded, users might need to refresh their browsers to experience any new features.
 
+.. note::
+
+  We only support a one minor version difference between the server versions when performing a rolling upgrade (for example v5.27.1 + v5.27.2 or v5.26.4 + v5.27.1 is supported, where as v5.25.5 + v5.27.0 is not supported). Running two different versions of Mattermost in your cluster should not be done outside of an upgrade scenario.
```
Comment (justinegeffen) [inline source/administration/upgrade.rst:135]:
```suggestion
  We only support a one minor version difference between the server versions when performing a rolling upgrade (for example v5.27.1 + v5.27.2 or v5.26.4 + v5.27.1 is supported, whereas v5.25.5 + v5.27.0 is not supported). Running two different versions of Mattermost in your cluster should not be done outside of an upgrade scenario.

```

File: source/deployment/cluster.rst [hunk 4]
```diff
@@ -343,6 +343,12 @@ A service interruption is not required for security patch dot releases of the Ma
 
 You can apply updates during a period when the anticipated load is small enough that one server can carry the full load of the system during the update.
 
+.. note::
+
+  .. note::
+
+  We only support a one minor version difference between the server versions when performing a rolling upgrade (for example v5.27.1 + v5.27.2 or v5.26.4 + v5.27.1 is supported, where as v5.25.5 + v5.27.0 is not supported). Running two different versions of Mattermost in your cluster should not be done outside of an upgrade scenario.
```
Comment (justinegeffen) [inline source/deployment/cluster.rst:350]:
```suggestion
  We only support a one minor version difference between the server versions when performing a rolling upgrade (for example v5.27.1 + v5.27.2 or v5.26.4 + v5.27.1 is supported, whereas v5.25.5 + v5.27.0 is not supported). Running two different versions of Mattermost in your cluster should not be done outside of an upgrade scenario.
```

File: source/deployment/cluster.rst [hunk 5]
```diff
@@ -343,6 +343,12 @@ A service interruption is not required for security patch dot releases of the Ma
 
 You can apply updates during a period when the anticipated load is small enough that one server can carry the full load of the system during the update.
 
+.. note::
+
+  .. note::
```
Comment (justinegeffen) [inline source/deployment/cluster.rst:348]: I think this can be removed as it's a duplicate.

**Review by grundleborg — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**